README.md

```diff
@@ -22,7 +22,7 @@ In this website you can add books of your choice to the list of books and save t
 
```

Comment:
- [ ] In keeping with ES6 syntax kindly refactor your functions to be arrow functions.

  BookList.js | Book.js
  :---:|:---:
  ![silvia-1](https://user-images.githubusercontent.com/36542986/171033047-336b1d11-3f18-4d3e-b920-c2ab92297317.jpg) | ![silvia-2](https://user-images.githubusercontent.com/36542986/171033054-0f452382-9470-4732-9121-6a5e0bf0a189.jpg)

Reply:
ok I will thanks

Reply:
please kindly check here I already use the arrow function
![image](https://user-images.githubusercontent.com/100236377/171049246-7434b2fb-c6fd-481e-a5af-05343f4921a0.png)

Reply:
@silviatofana yeah. You used arrow functions but it is expected that ALL your functions in your files are arrow functions.

Reply:
in all module files or in all js files ?

Reply:
@silviatofana all JS files that are relevant to this particular project.

Reply:
Whether in the `modules` folder or outside

Reply:
![image](https://user-images.githubusercontent.com/100236377/171052495-e52af658-5913-46df-a875-35649b2d9c62.png)